cpg-language-configfiles/src/test/resources/config.ini

```diff
@@ -0,0 +1,12 @@
+; Wikipedia example https://en.wikipedia.org/wiki/INI_file
```

Comment:
what's the license of the wikipedia file? Maybe just better to have our own example?

Reply:
Good point. Done.